src/components/Admin/ManageSensors/DownloadData/DownloadCSVButton.tsx

```diff
@@ -70,21 +74,23 @@ const DownloadCSVButton: ({
   ) {
     setFetchingData(true);
     const sensorsRef = firestore.collection('sensors');
-    let sensorDocs;
+    let sensorDocs: (
+      | firebase.firestore.QueryDocumentSnapshot<firebase.firestore.DocumentData>
```

Comment:
What does the extra preceding bar on line 78 do?

Reply:
Both lines are too long on their own, it's what the formatter did automatically